requirements-tests.txt

```diff
@@ -2,7 +2,7 @@ pep8
 flake8
 pylint
 mock
-coverage
+coverage<4
```

Comment:
I also had problems with the latest Coverage, it wasn't forwarding output correctly. What was the issue here?